### PR TITLE
[HAR-39] Adds checkbox to harbour

### PR DESCRIPTION
**Pull request by bartvandebiezen**

### CODE REVIEW

**Comment by stevengunneweg** [comment]
Ik mis de docs voor link (link hoort eig een eigen PR te zijn)

**Comment by bartvandebiezen** [comment]
@stevengunneweg The link component is essential for a complete and correct working checkbox component. For a more detailed link component including documentation I suggest a different pull request.

File: site/docs/controls/checkbox.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+---
+layout: docs
+title: Checkbox
+---
+
+## HTML structure
+```
+checkbox (checkbox--colored checkbox--stretched checkbox--inlined checkbox--90 checkbox--padded-0 checkbox--padded-90 checkbox--padded-100 checkbox--padded-200)
```
Comment (edgargressie) [inline site/docs/controls/checkbox.md]: I don't like the modifiers on a single line. The expanded version with described modifiers is more readable. See the popover.md for an example.

File: site/docs/controls/checkbox.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+	>
+	<span class="checkbox__label">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</span>
+</label>
+{% endcapture %}
+{% include example.html
+	content=checkbox
+%}
+
+## Checkbox 90
+{% capture checkbox %}
```
Comment (michaelmaat) [inline site/docs/controls/checkbox.md]: Reusing a variable name works., but maybe give this example its own unique name. e.g. checkbox-90

File: site/docs/controls/checkbox.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+{% capture multiple-checkboxes %}
+<label class="checkbox">
+	<input class="checkbox__input"
+		id="option-1"
+		name="option-1"
+		type="checkbox"
+		value="true"
+	>
+	<span class="checkbox__label">
+		Optie 1
```
Comment (romypetitjean) [inline site/docs/controls/checkbox.md]: Translate to English?

Comment (michaelmaat) [reply site/docs/controls/checkbox.md]: Fixed👮‍♂️

**Comment by michaelmaat** [comment]
> Unfortunately, this can result in unwanted spaces.

Did a visual check in the browser. Everything works now.

Only the `</a>` gave some problems, so did not place that one on a new line.

**Pull request merged**